tripleo-quickstart-config/roles/common/defaults/main.yml

```diff
@@ -178,9 +178,6 @@ networks:
           port: 2380
           target: "{{ cluster_name }}-etcd-2.{{ base_domain }}"
       hosts:
-        - ip: "{{ baremetal_network_cidr | nthhost(5) }}"
-          hostnames:
-            - "{{ cluster_name }}-api"
```

Comment:
This makes the API not be a VIP anymore.

Reply:
Right, it breaks master setup, needs to be reworked